test/e2e/rollback_test.go

```diff
@@ -26,11 +49,43 @@ var _ = Describe("rollback", func() {
 			for _, node := range nodes {
 				By(fmt.Sprintf("Check that %s has being rolled back", bridge1))
 				interfacesNameForNodeEventually(node).ShouldNot(ContainElement(bridge1))
-				By("Check reconcile re-apply desiredState")
+				By("Check that desiredState is applied")
 				interfacesNameForNodeEventually(node).Should(ContainElement(bridge1))
 				By(fmt.Sprintf("Check that %s is rolled back again", bridge1))
 				interfacesNameForNodeEventually(node).ShouldNot(ContainElement(bridge1))
 			}
 		})
 	})
+	Context("when connectivity to default gw is lost after state configuration", func() {
+		BeforeEach(func() {
+			By("Configure a invalid default gw")
+			for _, node := range nodes {
+				var address string
+				Eventually(func() string {
+					address = ipv4Address(node, "eth0")
+					return address
+				}, ReadTimeout, ReadInterval).ShouldNot(BeEmpty())
+				updateDesiredStateAtNode(node, badDefaultGw(address, "eth0"))
+			}
+		})
+		AfterEach(func() {
+			By("Clean up desired state")
+			resetDesiredStateForNodes()
+		})
+		It("should rollback to a good gw configuration", func() {
+			for _, node := range nodes {
+
+				By("Check that desiredState is applied")
+				Eventually(func() bool {
+					return dhcpFlag(node, "eth0")
+				}, ReadTimeout, ReadInterval).Should(BeFalse())
+
+				By(fmt.Sprintf("Check that %s is rolled back", "eth0"))
+				Eventually(func() bool {
+					return dhcpFlag(node, "eth0")
+				}, ReadTimeout, ReadInterval).Should(BeTrue())
+
```

Comment:
please drop this line. it is already indented, we don't need to divide it even more